chain-spec-generator/Cargo.toml

```diff
@@ -40,8 +39,9 @@ bridge-hub-kusama-runtime = { path = "../system-parachains/bridge-hubs/bridge-hu
 encointer-kusama-runtime = { path = "../system-parachains/encointer" }
 glutton-kusama-runtime = { path = "../system-parachains/gluttons/glutton-kusama" }
 
-# encointer is not migrated to the latest release, so we use compatible deps, when needed
-sp-core-encointer-compatible = { package = "sp-core",  version = "25.0.0" }
+# Encointer is(could be) not migrated to the latest release, so we use compatible deps, when needed
+# (if so, uncomment and set this version according to the actual encointer + check `fn encointer_kusama_genesis`)
+# sp-core-encointer-compatible = { package = "sp-core",  version = "29.0.0" }
```

Comment:
Can you open an issue for this please?

Reply:
I am in the middle of the rebase of bump 1.7.0 https://github.com/polkadot-fellows/runtimes/pull/187, probably I will need it there, if not I will remove this.

But this could be a potential problem for next releases again and again. E.g. if we bump `polkadot-sdk` deps and Encointer has different sp-core version, it won't compile here. Once I am done with fixing conflicts and rebasing 1.7.0, I will check, if we could fix it somehow and avoid this hach, if not I will create issue.

Reply:
We will need to find a way for Encointer's upgrades to be async with the other system chains. It is in Encointer's interest to stay up to date with other system chains and the sdk. I think we can commit to bumping our pallets within 2-3 weeks after a relevant polkadot-sdk upgrade meant to trigger a runtimes upgrade, but we might still need temporary workarounds

Reply:
I think that this hack is just because of AuraConfig, but if we create function in encointer runtime `fn aura_config_for_chain_spec(seed) -> AuraConfig`, we won't need this hack, I will try that

Reply:
Yeah generally Encointer should be open to update their stuff async in their own speed.

Reply:
I removed this hack for Encointer and fixed it, so we can bump Encointer independently.

https://github.com/polkadot-fellows/runtimes/pull/159/commits/bbf7c5bde3aad2b5b97b18744068e89d6925d9ae